test/Microsoft.AspNetCore.Server.Kestrel.FunctionalTests/RequestTests.cs

```diff
@@ -520,6 +521,53 @@ public void CanUpgradeRequestWithConnectionKeepAliveUpgradeHeader()
             }
         }
 
+        [Theory]
```

Comment:
> there's no PathBase left anywhere else but Frame

IIUC we would never set or get PathBase on Frame, right? Could we make the implementation of IHttpRequestFeature.PathBase in Frame.FeatureCollection.cs return string.Empty and throw on set?